config/default.schema.json

```diff
@@ -1240,7 +1133,7 @@
 		},
 		"FEATURE_ALLOW_INSECURE_LDAP_URL_ENABLED": {
 			"type": "boolean",
-			"default": false,
+			"default": true,
```

Comment:
you're right, forgot to change the value back, needed it for local testing.
formating is basically according to linter, so that can stay, even if nothing else is done to the file